packages/web-app-files/src/views/spaces/GenericSpace.vue

```diff
@@ -206,7 +206,7 @@ import ListInfo from '../../components/FilesList/ListInfo.vue'
 import NotFoundMessage from '../../components/FilesList/NotFoundMessage.vue'
 import QuickActions from '../../components/FilesList/QuickActions.vue'
 import ResourceDetails from '../../components/FilesList/ResourceDetails.vue'
-import ResourceTiles from '../../components/FilesList/ResourceTiles.vue'
+import ResourceTiles from '@ownclouders/web-pkg/src/components/FilesList/ResourceTiles.vue'
```

Comment:
```suggestion
import { ResourceTiles } from '@ownclouders/web-pkg'
```

You need to export `ResourceTiles` properly in `web-pkg` as well here.